tests/graphviz.txt

```diff
@@ -5,8 +5,9 @@ findimports can produce graph descriptions in the dot language used by
 graphviz
 
     >>> from findimports import main
-    >>> exitcode = main(['findimports', '-d', sample_tree])
+    >>> exitcode = main(['findimports', '-d', sample_tree, '--attrs', '{"rankdir": "TB"}'])
     digraph ModuleDependencies {
+      rankdir=TB
```

Comment:
Shouldn't there be a `;` at the end of this line?

Reply:
As per documentation attributes has no semicolon

Reply:
But the grammar at https://graphviz.org/doc/info/lang.html shows that a graph has a `;`-separated statement list inside the `{` `}`, and `ID = ID` is just one possible form of a statement.

I admit that I only checked the grammar and haven't actually tried running `dot` on any of the example graphs.

Reply:
I have tried it now, and dot from graphviz 2.43.0 accepts both spellings (with `;` and without `;`), and then rewrites them both into `graph [rankdir=TB];`

Oh!  I misread the grammar!  The `;` separator in the statement list is optional everywhere!

Still, since findimports adds `;` after nodes and edges, I'd like it to add `;` after attributes as well, for consistency.